### ISSUE TITLE
Link-Window cutted if Mattermost-Desktop-Client (Version 4.7.0) is maximized

### ISSUE THREAD

**Issue by DVTrast** [mattermost/desktop#1640]
I confirm (by marking "x" in the [ ] below: [x]):

- [x] This is not a troubleshooting question. [Troubleshooting questions go here: http://www.mattermost.org/troubleshoot/](http://www.mattermost.org/troubleshoot/).
- [x] This doesn't reproduce on web browsers (such as in Chrome). If it does, [issue reports go to the Mattermost Server repository](https://github.com/mattermost/platform/issues).
- [x] I have read [contributing guidelines](https://github.com/mattermost/desktop/blob/master/CONTRIBUTING.md).

---

**Summary**

**Environment**
- Operating System: Windows 10 Pro 1909
- Mattermost Desktop App version: 4.7.0 
- Mattermost Server version: 5.36.1 

**Steps to reproduce**
Open the Mattermost-Desktop-Client an maximize it (fullscreen). Paste a link to a channel and mouse over it, so on the bottom left the link-window will be shown.
**Expected behavior**
The whole link-windows should be shown.
**Observed behavior**
You can only see the top of the link-window, because the rest is cutted from the taskbar respectively the monitor.

Link-Window cutted by monitor:
![link_window_cutted_by_monitor](https://user-images.githubusercontent.com/61227869/123747640-7a727880-d8b3-11eb-8acf-4067da969a1c.png)

Link-Window cutted by taskbar:
![link_window_cutted_by_taskbar](https://user-images.githubusercontent.com/61227869/123747931-d937f200-d8b3-11eb-9811-05bc6834654f.png)

.

**Possible fixes**

**Comment by amyblais** [comment]
@jgilliam17 Can you reproduce this?

**Comment by jgilliam17** [comment]
@amyblais Yes, I can reproduce on Win 10, MSI build. I'll create a Jira ticket.

**Comment by jgilliam17** [comment]
https://mattermost.atlassian.net/browse/MM-36727

**Comment by DVTrast** [comment]
Is there actual a date, on which the Version 5 of the Mattermost Desktop-Client will be released?

**Comment by amyblais** [comment]
> Is there actual a date, on which the Version 5 of the Mattermost Desktop-Client will be released?

It is planned for September 15th together with v6.0 server release on that day.
